nifi-registry/nifi-registry-core/nifi-registry-resources/src/main/resources/conf/nifi-registry.properties

```diff
@@ -13,6 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+nifi.registry.version=${project.version}
```

Comment:
I do not recommend leaving the version value exposed in the properties file. This allows for the possibility of user modification which may not correspond to the actual version of the running NiFi Registry.

Reply:
Got it!
I understand that the nifi main interface has something similar and am looking into how it accomplishes the same functionality but without using a property like this.

If anyone else looking at this has some additional information, I'd love to hear it

Reply:
Found a way to derive it from the `NiFiRegistryApiApplication`